### PR TITLE
Remove Dark Theme & change into system theme and update previous code 

### PR DESCRIPTION
**Pull request by hafizsaadiftikhar**

### CODE REVIEW

**Comment by hank-milliken** [comment]
Hi 

1. please rebase the code so there are no merge commits
2. why is `.gradle` committed again? This is persisitn issue in ever PR you have made so far

File: .gitignore [hunk 1]
```diff
@@ -9,7 +11,6 @@
 
 *.iml
 .gradle
-.idea
```
Comment (hank-milliken) [inline .gitignore:12]: why?

Comment (hafizsaadiftikhar) [reply .gitignore:12]: The PR that is Submitted almost 10 days ago was included but at the latest, it was removed. It was submitted by mistake

**Comment by hafizsaadiftikhar** [comment]
Please resolve the conflicts and accept the latest PR. Test it according to the latest code. The design must be same as that was sent to you by using brand color

**Pull request closed without merging**